packages/calcite-components/src/components/select/select.scss

```diff
@@ -3,49 +3,54 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-select-font-size: The font size of `calcite-option`s in the component.
- * @prop --calcite-select-spacing: The padding around the selected option text.
- * @prop --calcite-select-text-color: The text color of the component.
+ * @prop --calcite-select-font-size: Specifies the font size of `calcite-option`s in the component.
+ * @prop --calcite-select-text-weight: Specifies the font weight of `calcite-option`s in the component.
+ * @prop --calcite-select-spacing-inline: Specifies the inline padding of `calcite-option`s in the component.
+ * @prop --calcite-select-text-color: Specifies the text color of `calcite-option`s in the component.
+ * @prop --calcite-select-size: Specifies the component's width.
+ * @prop --calcite-select-block-size: Specifies the component's height.
```

Comment:
I don't think we need this one, as the user can specify this on the element itself.